app/views/components/_option-select.html.erb

```diff
@@ -6,6 +6,8 @@
   <div class="container-head js-container-head">
     <div class="option-select-label" id="<%= title_id %>">
       <%= title %>
+      <svg version="1.1" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg" class="app-c-option-select__icon app-c-option-select__icon--up"><path d="m798.16 609.84l-256-256c-16.683-16.683-43.691-16.683-60.331 0l-256 256c-16.683 16.683-16.683 43.691 0 60.331s43.691 16.683 60.331 0l225.84-225.84 225.84 225.84c16.683 16.683 43.691 16.683 60.331 0s16.683-43.691 0-60.331z"/></svg>
+      <svg version="1.1" viewBox="0 0 1024 1024" xmlns="http://www.w3.org/2000/svg" class="app-c-option-select__icon app-c-option-select__icon--down"><path d="m225.84 414.16l256 256c16.683 16.683 43.691 16.683 60.331 0l256-256c16.683-16.683 16.683-43.691 0-60.331s-43.691-16.683-60.331 0l-225.84 225.84-225.84-225.84c-16.683-16.683-43.691-16.683-60.331 0s-16.683 43.691 0 60.331z"/></svg>
```

Comment:
Do we need svg? Could we not replace it with a css-only version using `:after` pseudo element?
GOV.UK Design System uses something similar for the "Back link" icon and this should have better browser support than using svg.

https://design-system.service.gov.uk/components/back-link/default/index.html
https://design-system.service.gov.uk/components/back-link/
https://css-tricks.com/snippets/css/css-triangle/

Reply:
We could, but technically they're a hack and making sure they work properly in all the browsers we want can be fiddly. I'd rather use SVG here because I know it will either work or not appear, and the support is more than we need.